tests/test_write_nwb.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+import datetime
+import pynwb
+import h5py
+
+from ipfx.x_to_nwb.NWBConverter import NWBConverter
+from .helpers_for_tests import diff_h5, validate_nwb
+from ipfx.bin.run_nwb1_to_nwb2_conversion import make_nwb2_file_name
+from hdmf import Container
+import numpy as np
+import ipfx.nwb_reader as nwb_reader
+
+from ipfx.bin.run_feature_extraction import embed_spike_times
+
+
+
+
+
+def make_skeleton_nwb1_file(nwb1_file_name):
+
+    with h5py.File(nwb1_file_name, 'w') as fh:
+        dt = h5py.special_dtype(vlen=bytes)
+        dset = fh.create_dataset("nwb_version", (1,), dtype=dt)
+        dset[:] = "NWB-1"
+        fh.create_group("acquisition/timeseries")
+        fh.create_group("analysis")
+
+
+def make_skeleton_nwb2_file(nwb2_file_name):
+
+    nwbfile = pynwb.NWBFile(
+        session_description='test icephys',
+        identifier='session_uuid',
+        session_start_time=datetime.datetime.now(),
+        file_create_date=datetime.datetime.now()
+    )
+
+    device = nwbfile.create_device(name='electrode_0')
+    electrode = nwbfile.create_ic_electrode(name="elec0",
+                                            description='intracellular electrode',
+                                            device=device)
+
+    io = pynwb.NWBHDF5IO(nwb2_file_name, 'w')
+    io.write(nwbfile)
+    io.close()
+
+
+
+def test_embed_spike_times_into_nwb2(tmpdir_factory):
+
+    sweep_spike_times = {
```

Comment:
I would just have this at the module level since it's reused in both tests without changes